packages/language-server/index.js

```diff
@@ -316,6 +317,11 @@ documents.onDidClose((event) => {
  */
 async function checkDiagnostics(event) {
   const {uri} = event.document
+
+  if (!isMdx(uri)) {
+    return
```

Comment:
I’m not sure how to test this. Explicitly no diagnostics should be sent, meaning we should basically assert a promise never resolves.

Reply:
I’ll get back to this when I add code coverage checks. For now I just really want to release these changes.